html/index.html

```diff
@@ -142,12 +142,13 @@ <h3 class="title is-size-4-desktop is-size-5">Enter address and amount (0.001-0.
                                         <img id="captcha_img" src="/captcha?<%- new Date().getTime() %>"/>
                                     </div>
                                     <div class="control">
-                                        <input class="input" id="captcha" name="captcha" type="input" placeholder="captcha">
+                                        <input class="input" id="captcha" name="captcha" type="input" placeholder="captcha" tabindex="3">
                                     </div>
                                 </div>
                             <% } %>
 
-                            <button id="sendButton" onclick="javascript:faucet_fetch();" class="button is-primary is-fullwidth has-margin-top-25">Send</button>
+                            <button id="sendButton" class="button is-primary is-fullwidth has-margin-top-25 tabindex="4"">Send</button>
```

Comment:
... and this 5.

Reply:
Hmm, I thought that only one of `password` or `captcha` is shown…

Reply:
Good point, I believe you're correct.

Reply:
@kallewoof I would still change the ordering, so that SEND button is tabstop 3 and those `password`/`captcha` are 4 (since when they are not shown it is bad UX).